wcfsetup/install/files/lib/system/template/TemplateScriptingCompiler.class.php

```diff
@@ -689,19 +689,27 @@ protected function compileForeachTag($foreachTag) {
 			$foreachProp = "\$this->v['tpl']['foreach'][".$args['name']."]";
 		}
 		
+		$hash = StringUtil::getRandomID();
```

Comment:
Is there a particular reason behind using `$hash` and `"\$_foreach_".$hash` everywhere instead of using something like this:
```
$foreachHash = '$_foreach_' . StringUtil::getRandomID();
```
After all, it's always the same variabl ename, and the constant string concats make the code a bit harder to read than it should be.